composables/useMultipleBalance.ts

```diff
@@ -164,15 +157,15 @@ export default function (refetchPeriodically: boolean = false) {
           chain,
           tokenId,
         }),
-      EVM: () => getEvmBalance({ address: currentAddress as Address, prefix }),
+      EVM: () => getEvmBalance({ address: currentAddress }),
```

Comment:
how would you know which chain balance you should get?

Reply:
updated on `useBalance` side instead